apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -13,3 +13,46 @@
     }
   }
 }
+
+.reaction-recorder {
+  .webcam {
+    width: 15%;
+
+    video {
+      width: 100%;
+    }
+
+    &.offline {
+      video {
+        display: none;
+      }
+    }
+
+    &::after {
+      content: ' ';
+      display: block;
+      visibility: hidden;
+      background-color: red;
```

Comment:
The link explains it: "Color keywords look like variables but are not variables." which is fair enough, I suppose.